pkg/azuredisk/nodeserver.go

```diff
@@ -81,7 +82,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 	}
 	defer d.volumeLocks.Release(diskURI)
 
-	lun, ok := req.PublishContext[LUN]
+	lun, ok := req.PublishContext[azureutils.LUN]
```

Comment:
`consts.LUN` ?

Reply:
Since this file uses utilities functions from `azureutils`, I have imported the package as utilities instead of `consts`. But I will be separating the two to have a constants package and can then make this change.